aten/src/ATen/native/quantized/cpu/qembeddingbag_prepack.cpp

```diff
@@ -133,10 +134,73 @@ Tensor qembeddingbag_4bit_prepack(const Tensor& weight) {
   return output;
 }
 
+/*
+ * Prepack function for embedding_bag weights.
+ * This function expects a per-row quantized weight tensor
+ * with a floating point scale and zero_point value.
+ * zero point is set to be (-Xmin/scale)
+ * To prepack the weights we store the scale and bias (where bias is Xmin)
+ * for each row along with the quantized weights.
+ */
+// TODO: Extend this to support 4-bits once 4-bit qtensor support is added.
+Tensor qembeddingbag_prepack(at::Tensor qweight) {
+  Tensor weight_contig = qweight.contiguous(qweight.suggest_memory_format());
+  const uint8_t* weight_data =
```

Comment:
do we want to check the dtype here?

Reply:
Sure, I can add a check. We don't check dtype in many ops though, probably because `data_ptr` would error out for incorrect type